### PR TITLE
Serial dilution fix

### PR DESCRIPTION
**Pull request by jarpinosynthace**

### CODE REVIEW

File: an/Liquid_handling/SerialDilution/SerialDilution.an [hunk 1]
```diff
@@ -33,6 +33,7 @@ Inputs (
 // Physical outputs from this protocol with types
 Outputs (
 	Dilutions []*LHComponent
+	Waste *LHComponent
```
Comment (chr1sgrant) [inline an/Liquid_handling/SerialDilution/SerialDilution.an]: Could this be an option to dispose of the excess? 
The user might not care about the excess volume or want to waste an extra plate/ have room for an extra plate on deck

Comment (chr1sgrant) [inline an/Liquid_handling/SerialDilution/SerialDilution.an]: I think exporting the final concentrations would be desirable too

Comment (jarpinosynthace) [inline an/Liquid_handling/SerialDilution/SerialDilution.an]: Yes this is for disposing of the excess, I can add it as a bool option if people want it removed or not but in most cases i would imagine people would. Like we were saying the other day people like all the volumes to be the same at the end of a dilution series

File: an/Liquid_handling/SerialDilution/SerialDilution.an [hunk 2]
```diff
@@ -5,34 +5,47 @@ protocol SerialDilution
...
+	//The physical plate where your serial dilutions will be made.
 	OutPlate *LHPlate
 )
 
 // Physical outputs from this protocol with types
 Outputs (
+	// The physical dilutions made by this element.
 	Dilutions []*LHComponent
+	// The excess solution removed from the final dilution, this is moved to the input plate.
+	Waste *LHComponent
```
Comment (chr1sgrant) [inline an/Liquid_handling/SerialDilution/SerialDilution.an]: You don't need to export the waste

Comment (chr1sgrant) [inline an/Liquid_handling/SerialDilution/SerialDilution.an]: So where will the waste mix to now anyway? there's no plate specified so might mix onto the input plate or dilution plate at next available well.

**Pull request merged**